frontend/src/components/single-choice/single-choice.tsx

```diff
@@ -8,19 +8,23 @@ import { OptionSingleChoice } from './single-choice.styles';
 export interface SingleChoiceProps {
   options: OptionQuestion[];
   setResult: Dispatch<SetStateAction<number[]>>;
+  current_answers_survey_id?: number;
+  question_id?: number;
 }
 
 const SingleChoice = (props: SingleChoiceProps) => {
-  const { options, setResult } = props;
+  const { options, setResult, current_answers_survey_id, question_id } = props;
   const answerService = new AnswerService(httpClient);
 
   const selectOption = async (id: number) => {
-    await answerService.create({
-      question_id: 853,
-      answers_survey_id: 121,
-      user_answer: 'student@gmail.com',
-      option_ids: [id],
-    });
+    if (current_answers_survey_id && question_id) {
```

Comment:
Do you have tests for when this if returns false?
If not, add them or remove the if

Reply:
Yep, has a test for this.

Reply:
Nice!!!